extensions/wikia/LinkSuggest/LinkSuggest.class.php

```diff
@@ -207,20 +203,33 @@ static function getLinkSuggest(WebRequest $request) {
 				$pageTitleLikeClause = "'{$queryLower}%'";
 			}
 
-			// TODO: use $db->select helper method
```

Comment:
This TODO has been here for 7 years...It probably thought it was long forgotten. Well done addressing it :)